### PR TITLE
feat: add antdevtools 0.20.1

### PR DESCRIPTION
**Pull request by alswl**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[closed issues]: https://github.com/Homebrew/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/antdevtools.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'antdevtools' do
+  version '0.20.1,63d9b84e-bbcf-4a00-9427-0bb3f713c769'
+  sha256 '4c9e207838ad0f7c6159d4ebd8c828d54b1aec28622a9f67f4f4bd0acf93664a'
+
+  # gw.alipayobjects.com was verified as official when first introduced to the cask
+  url "https://gw.alipayobjects.com/os/Ant-devtools-Release/#{version.after_comma}/#{version.major}_#{version.minor}_#{version.patch}/AntDevtools.dmg"
+  name '蚂蚁开发者工具'
```
Comment (vitorgalvao) [inline Casks/antdevtools.rb]: Another `name` stanza, with the english name, must come first.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/antdevtools.rb (hunk 1 above)
Comment (alswl) [reply Casks/antdevtools.rb]: fixed.

**Comment by vitorgalvao** [comment]
I can’t find a way to download this app without giving them an email address.

**Comment by alswl** [comment]
Try:

```bash
wget 'https://gw.alipayobjects.com/os/Ant-devtools-Release/63d9b84e-bbcf-4a00-9427-0bb3f713c769/0_20_1/AntDevtools.dmg'
```

**Comment by vitorgalvao** [comment]
@alswl That’s not the point. Since the hostname for `homepage` and `url` differ, [we need to verify it](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment). [And if we can’t get to said `url` without giving an email](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask), [we can’t accept the cask](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#unofficial-vendorless-and-walled-builds).

**Comment by alswl** [comment]
1.   open 'https://mini.open.alipay.com/channel/miniIndex.htm', (there was no English version now.)
2.  click the menubar: "开发", it means "develop"
3.  in the section '下载/登录 IDE', it means "download/login IDE", click "小程序开发 IDE 工具", it means ant small program IDE tools
4. in the section "Stable v0.20.1", click Mac
5. you will get url "http://p.tb.cn/Ant-devtools-Release_prod-mac/AntDevtools.dmg", then it will redirect to "https://gw.alipayobjects.com/os/Ant-devtools-Release/63d9b84e-bbcf-4a00-9427-0bb3f713c769/0_20_1/AntDevtools.dmg"
6. thanks for your patient

**Comment by vitorgalvao** [comment]
Confirmed. Thank you for the instructions and the submission.

**Pull request merged**